### ISSUE TITLE
Understanding 1.4.11 - symbolic text characters

### ISSUE THREAD

**Issue by fstrr** [w3c/wcag#2062]
Unless I'm missing something in the Understanding document, I think that clarification is needed for the level of contrast for components that only contain what Label In Name describes as [symbolic text characters](https://www.w3.org/WAI/WCAG21/Understanding/label-in-name.html#symbolic-text-characters). Taking the common example of a close button with only an X symbol in it, at an equivalent size of 16px:

1. `<button aria-label="close">X</button>` : does the letter X require a minimum contrast of 4.5:1 as it's text?
2. `<button aria-label="close"><svg aria-hidden="true" class="button-symbol">…</svg></button>`: does the SVG require a minimum contrast of 3:1 contrast as it's an image?

**Comment by patrickhlauke** [comment]
the "X" in this case is not text per se ... it's not representing the "letter X", but is used as an icon, so yes, 1.4.11 applies and it only normatively needs to be 3:1 contrast minimum

**Comment by JAWS-test** [comment]
I agree with @patrickhlauke, but think that an addition to Understanding in this regard would be useful

**Comment by patrickhlauke** [comment]
Made a first naive stab at it ... #2067
